recipes/QCF/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+cd $SRC_DIR
```

Comment:
should not be necessary

Reply:
Thanks for the comments!

I have tried to do the rebase procedure several times without success (I'm not really sure I fully understand how it works). I'm not sure what the issue is, but if you have any general suggestions I would very much appreciate it. Here is what I tried:

```
git checkout QCF
git rebase master
```

I have a few changes to make to my code as well, so would it be easier to make those changes and start a new submission?